packages/node_modules/@ciscospark/spark-core/src/interceptors/auth.js

```diff
@@ -61,14 +61,6 @@ export default class AuthInterceptor extends Interceptor {
       return Promise.resolve(false);
     }
 
-    // returns true if uri is in whitelistedServiceDomains and is requested
-    if (options.uri && options.runWhitelistedDomains) {
```

Comment:
this will be a breaking change for any dev that doesn't update their code, right?

Reply:
No if a developer doesn't pass the header and the global allow i false we dont add the header or do whitelisting check